src/libraries/System.Net.Http/src/System/Net/Http/Metrics/MetricsHandler.cs

```diff
@@ -0,0 +1,266 @@
+            KeyValuePair.Create<int, object>(226, 226),
+
+            KeyValuePair.Create<int, object>(300, 300),
+            KeyValuePair.Create<int, object>(301, 301),
+            KeyValuePair.Create<int, object>(302, 302),
+            KeyValuePair.Create<int, object>(303, 303),
+            KeyValuePair.Create<int, object>(304, 304),
+            KeyValuePair.Create<int, object>(305, 305),
+            KeyValuePair.Create<int, object>(306, 306),
+            KeyValuePair.Create<int, object>(307, 307),
+            KeyValuePair.Create<int, object>(308, 308),
+
+            KeyValuePair.Create<int, object>(400, 400),
+            KeyValuePair.Create<int, object>(401, 401),
+            KeyValuePair.Create<int, object>(402, 402),
+            KeyValuePair.Create<int, object>(403, 403),
+            KeyValuePair.Create<int, object>(404, 404),
+            KeyValuePair.Create<int, object>(405, 405),
+            KeyValuePair.Create<int, object>(406, 406),
+            KeyValuePair.Create<int, object>(407, 407),
+            KeyValuePair.Create<int, object>(408, 408),
+            KeyValuePair.Create<int, object>(409, 409),
+            KeyValuePair.Create<int, object>(410, 410),
+            KeyValuePair.Create<int, object>(411, 411),
+            KeyValuePair.Create<int, object>(412, 412),
+            KeyValuePair.Create<int, object>(413, 413),
+            KeyValuePair.Create<int, object>(414, 414),
+            KeyValuePair.Create<int, object>(415, 415),
+            KeyValuePair.Create<int, object>(416, 416),
+            KeyValuePair.Create<int, object>(417, 417),
+            KeyValuePair.Create<int, object>(418, 418),
+            KeyValuePair.Create<int, object>(419, 419),
+            KeyValuePair.Create<int, object>(421, 421),
+            KeyValuePair.Create<int, object>(422, 422),
+            KeyValuePair.Create<int, object>(423, 423),
+            KeyValuePair.Create<int, object>(424, 424),
+            KeyValuePair.Create<int, object>(426, 426),
+            KeyValuePair.Create<int, object>(428, 428),
+            KeyValuePair.Create<int, object>(429, 429),
+            KeyValuePair.Create<int, object>(431, 431),
+            KeyValuePair.Create<int, object>(451, 451),
+            KeyValuePair.Create<int, object>(499, 499),
+
+            KeyValuePair.Create<int, object>(500, 500),
+            KeyValuePair.Create<int, object>(501, 501),
+            KeyValuePair.Create<int, object>(502, 502),
+            KeyValuePair.Create<int, object>(503, 503),
+            KeyValuePair.Create<int, object>(504, 504),
+            KeyValuePair.Create<int, object>(505, 505),
+            KeyValuePair.Create<int, object>(506, 506),
+            KeyValuePair.Create<int, object>(507, 507),
+            KeyValuePair.Create<int, object>(508, 508),
+            KeyValuePair.Create<int, object>(510, 510),
+            KeyValuePair.Create<int, object>(511, 511)
+        });
+
+        private sealed class SharedMeter : Meter
```

Comment:
This derived class is no longer necessary I think? SharedMeter.Instance is never exposed to user code.

Reply:
It can be still disposed by `MeterListner.InstrumentPublished` subscribers. I know if it falls into the "just don't do that" category, but if we can protect against it with a few lines of code, then why not?

Reply:
Ah, yes it does get exposed that way. Its not a pattern I expect anyone else to follow, but agree it causes no problems.